lib/model/store.dart

```diff
@@ -267,6 +275,10 @@ class PerAccountStore extends ChangeNotifier {
         } else {
           profileData.remove(update.id);
         }
+        if (profileData.isEmpty) {
+          // null is equivalent to `{}` for efficiency; see [User._readProfileData].
+          user.profileData = null;
```

Comment:
nit: in commit message:

    store: ensure empty map in profileData updates are converted to null

use sentence case for summary (i.e., s/ensure empty/Ensure empty/)

Reply:
bump